src/layouts/components/Homepage/HeroHighlightSection.tsx

```diff
@@ -0,0 +1,193 @@
+            />
+            {/* TODO: Validate button.  
+              This isn't being done on prod also.
+             */}
+            <FormErrorMessage>{errors.button}</FormErrorMessage>
+          </FormControl>
+          <FormControl isRequired isInvalid={!!errors.url}>
+            <FormLabel>Button URL</FormLabel>
+            <Input
+              placeholder="Insert /page-url or https://"
+              value={url}
+              onChange={onChange}
+              id="section-0-hero-url"
+            />
+            <FormErrorMessage>{errors.url}</FormErrorMessage>
+          </FormControl>
+        </VStack>
+        <Text mt="1.5rem" textStyle="h6">
+          Highlights
+        </Text>
+        <Text mt="0.5rem" textStyle="body-2" textColor="base.content.medium">
+          Drag and drop highlights to rearrange them. On a desktop screen,
+          highlights are shown side-by-side
+        </Text>
+        <Editable.Accordion>
+          <Editable.EmptySection
+            isEmpty={highlights.length === 0}
+            title="Highlights you add will appear here"
+            subtitle="You can call out informative links using highlights"
+          >
+            <Editable.Section px={0} spacing="1.25rem" py="1.5rem">
+              {highlights.map(
+                (
+                  {
+                    title: highlightTitle,
+                    url: highlightUrl,
+                    description: highlightDescription,
+                  },
+                  highlightIndex
+                ) => {
+                  return (
+                    <Editable.DraggableAccordionItem
+                      title={highlightTitle || "New highlight"}
+                      draggableId={`highlight-${highlightIndex}-draggable`}
+                      index={highlightIndex}
+                      isInvalid={_.some(errors.highlights[highlightIndex])}
+                    >
+                      <Editable.Section>
+                        <FormControl
+                          isInvalid={!!errors.highlights[highlightIndex].title}
+                          isRequired
+                        >
+                          <FormLabel>Title</FormLabel>
+                          <Input
+                            placeholder="Highlight title"
+                            id={`highlight-${highlightIndex}-title`}
+                            value={highlightTitle}
+                            onChange={onChange}
+                          />
+                          <FormErrorMessage>
+                            {errors.highlights[highlightIndex].title}
+                          </FormErrorMessage>
+                        </FormControl>
+                        <FormControl
+                          isInvalid={
+                            !!errors.highlights[highlightIndex].description
+                          }
+                          isRequired
+                        >
+                          <FormLabel>Title</FormLabel>
```

Comment:
The title of this section should be `Description` instead of `Title` according to the design.

![image](https://github.com/isomerpages/isomercms-frontend/assets/14878879/66595954-3f6e-4d90-b6c8-a32d17951a10)

Reply:
missed this, fixed!